gulpfile.js

```diff
@@ -19,7 +19,8 @@ var PATHS = {
   specs: 'src/**/*.spec.ts',
   demo: 'demo/**/*.ts',
   demoDist: 'demo/dist/**/*',
-  typings: 'typings/browser.d.ts'
+  typings: 'typings/browser.d.ts',
+  demoDocsJson: 'demo/src/docs.json'
```

Comment:
Could we move it to another folder, something that is specifically not under 'src'? Like `demo/temp` or `demo/generated`? This way people won't be tempted to generate + edit + change `.gitignore`.